Sources/SwiftGenKit/Utils/YAML.swift

```diff
@@ -61,3 +61,50 @@ public enum YAML {
     }
   }
 }
+
+// Copied from /Source/SwiftLintFramework/Models/YamlParser.swift
+// of https://github.com/realm/SwiftLint/blob/d1dbc31aa9269364d6d7f43d2f99c82e12ceca6f
```

Comment:
I think you can directly link to the file at a given commit in one link: https://github.com/marcelofabri/SwiftLint/blob/33ca7d71f77025100dcb4bab5f464d342664cc5b/Source/SwiftLintFramework/Models/YamlParser.swift